### ISSUE TITLE
Feature parity with Umami's tracker

### ISSUE THREAD

**Issue by ijkml** [ijkml/umami-tracker#1]
Let's call Umami's tracker `Tracker-01` and this one `Tracker-99` (I know, I know, I'm open to suggestions :laughing:)

To start, `Tracker-99` (if and when it ever comes to completion) might and should not be a drop-in replacement (alternative) to `Tracker-01`. Partly because `Tracker-01` aims to be as simple as possible for the end user: just copy the code from your dashboard and paste in your `<head>`. `Tracker-99` aims to be more programmatic.

So, there are a few changes (already), like: 
- minor alteration to the script, dropping support for IE `do-not-track` that relied on `Window.external` which is deprecated.
- CSS events dropped. Programmatic, remember?
- No auto-track. Prog... okay, you get. 

See: https://github.com/ijkml/umami-tracker/blob/main/DIFF.md

**Comment by ijkml** [comment]
Hey, @vencho-mdp :wave:  would you be interested in helping flesh this out?

**Comment by vencho-mdp** [comment]
Absolutely! Should I check the issues opened in that repo?

**Comment by ijkml** [comment]
Yes, thank you!

**Comment by vencho-mdp** [comment]
I think it would be nice to use the `nuxt-umami` as a wrapper for this package for Nuxt. In that way, we're not closing opportunities to integrate with other fws. What's your take?

**Comment by ijkml** [comment]
Yeah, yeah, that's what I had in mind. 👍 This package should be plain, independent. Nuxt-Umami would wrap it.
